biojava-structure/src/main/java/org/biojava/nbio/structure/cluster/SubunitClustererParameters.java

```diff
@@ -39,13 +41,62 @@
 	private int absoluteMinimumSequenceLength = 5;
 	private double minimumSequenceLengthFraction = 0.75;
 
-	private double sequenceIdentityThreshold = 0.95;
+	// "local" metrics are separately scoring
+	// SubunitClustererMethod.SEQUENCE: sequence identity of a local alignment (normalised by the number of
+	// aligned residues) and sequence coverage of the alignment
+	// SubunitClustererMethod.STRUCTURE: RMSD of the aligned substructures and structure coverage of the alignment
+	// require two respective thresholds for each method
+
+	// "global" metrics are scoring
+	// SubunitClustererMethod.SEQUENCE: sequence identity of a global alignment (normalised by the number of columns)
+	// SubunitClustererMethod.STRUCTURE: TMScore of the aligned structures (normalised by the larger structure)
+	// require one threshold for each method
+	private boolean useGlobalMetrics;
+	private double sequenceIdentityThreshold;
+	private double sequenceCoverageThreshold = 0.75;
+
 	private double rmsdThreshold = 3.0;
-	private double coverageThreshold = 0.75;
+	private double structureCoverageThreshold = 0.75;
+	private double tmThreshold = 0.5;
 
 	private SubunitClustererMethod clustererMethod = SubunitClustererMethod.STRUCTURE;
+
+	private String superpositionAlgorithm = CeMain.algorithmName;
+	private boolean optimizeAlignment = true;
+
+	private boolean useSequenceCoverage;
+	private boolean useRMSD;
+	private boolean useStructureCoverage;
+	private boolean useTMScore;
+
 	private boolean internalSymmetry = false;
 
+	// subunits aligned with these or better scores will be considered "identical"
+	private static final double hcSequenceIdentityLocal = 0.95;
+	private static final double hcSequenceCoverageLocal = 0.75;
+	private static final double hcSequenceIdentityGlobal = 0.85;
+
+	public SubunitClustererParameters() {
+		this(false); // initialize with local metrics by default
+	}
+
+	public SubunitClustererParameters(boolean iUseGlobalMetrics) {
+		useGlobalMetrics = iUseGlobalMetrics;
```

Comment:
Docs for this would be important. Also the 2 variables can be called the same way and then use:
```java
this.useGlobalMetrix=useGlobalMetrics;
```